updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -18,6 +18,7 @@ There are a number of caveats to consider when attempting an EUS-to-EUS update.
 * You must be running at least {product-title} 4.8.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.8.z before attempting the EUS-to-EUS update.
 * Support for RHEL7 workers was removed in {product-title} 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers.
 * Node components are not updated to {product-title} 4.9. Do not expect all features and bugs fixed in {product-title} 4.9 to be made available until you complete the update to {product-title} 4.10 and enable all MachineConfigPools to update.
+* EUS-to-EUS updates are not applicable to compact 3 node clusters and Single Node Openshift (SNO) topologies.
```

Comment:
Feedback from trying to explain this to users is that the reason isn't obvious, even in the context of this document, so I think we need to expand this statement with more detail.

```suggestion
* EUS-to-EUS updates are not applicable to compact 3 node clusters and Single Node Openshift (SNO) topologies because all of the hosts in these configurations are master hosts.
```

Reply:
openshift/enhancements#1041 landed a few weeks back for single-node-control-plane-plus-compute.  So do we want to make a more generic statement like "this process is only applicable to clusters that have non-control-plane compute nodes"?